package.json

```diff
@@ -19,7 +19,9 @@
     "stylus": "~0.54.5",
     "uglify-js": "^2.7.5"
   },
-  "devDependencies": {},
+  "devDependencies": {
+    "jsonlint": "^1.6.2"
```

Comment:
You mean in the unlikely case `-q` is removed? When that happens, you just fix it. This is not part of a dependency that may silently introduce runtime behavior change of the application.